database/migrations/2021_12_27_211938_drop_score_from_ratings_table.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+class DropScoreFromRatingsTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::table('ratings', function (Blueprint $table) {
+            $table->dropColumn('score');
+        });
+    }
+
+    /**
+     * Reverse the migrations.
+     *
+     * @return void
+     */
+    public function down()
+    {
+        Schema::table('ratings', function (Blueprint $table) {
+            //
```

Comment:
migrations should reverse themselves :) 
```suggestion
            $table->string('score')
```